pkg/http/api_subtlecrypto.go

```diff
@@ -0,0 +1,136 @@
+		},
+		{
+			Methods: []string{fasthttp.MethodPost},
+			Route:   "subtlecrypto/{name}/decrypt",
+			Version: apiVersionV1alpha1,
+			Handler: a.onPostSubtleCryptoDecrypt(),
+		},
+		{
+			Methods: []string{fasthttp.MethodPost},
+			Route:   "subtlecrypto/{name}/wrapkey",
+			Version: apiVersionV1alpha1,
+			Handler: a.onPostSubtleCryptoWrapKey(),
+		},
+		{
+			Methods: []string{fasthttp.MethodPost},
+			Route:   "subtlecrypto/{name}/unwrapkey",
+			Version: apiVersionV1alpha1,
+			Handler: a.onPostSubtleCryptoUnwrapKey(),
+		},
+		{
+			Methods: []string{fasthttp.MethodPost},
+			Route:   "subtlecrypto/{name}/sign",
+			Version: apiVersionV1alpha1,
+			Handler: a.onPostSubtleCryptoSign(),
+		},
+		{
+			Methods: []string{fasthttp.MethodPost},
+			Route:   "subtlecrypto/{name}/verify",
+			Version: apiVersionV1alpha1,
+			Handler: a.onPostSubtleCryptoVerify(),
+		},
+	}
+}
+
+func (a *api) onPostSubtleCryptoGetKey() fasthttp.RequestHandler {
+	return UniversalFastHTTPHandler(
+		a.universal.SubtleGetKeyAlpha1,
+		UniversalFastHTTPHandlerOpts[*runtimev1pb.SubtleGetKeyAlpha1Request, *runtimev1pb.SubtleGetKeyAlpha1Response]{
+			InModifier: subtleCryptoInModifier[*runtimev1pb.SubtleGetKeyAlpha1Request],
+		},
+	)
+}
+
+func (a *api) onPostSubtleCryptoEncrypt() fasthttp.RequestHandler {
+	return UniversalFastHTTPHandler(
+		a.universal.SubtleEncryptAlpha1,
+		UniversalFastHTTPHandlerOpts[*runtimev1pb.SubtleEncryptAlpha1Request, *runtimev1pb.SubtleEncryptAlpha1Response]{
+			InModifier: subtleCryptoInModifier[*runtimev1pb.SubtleEncryptAlpha1Request],
+		},
+	)
+}
+
+func (a *api) onPostSubtleCryptoDecrypt() fasthttp.RequestHandler {
+	return UniversalFastHTTPHandler(
+		a.universal.SubtleDecryptAlpha1,
+		UniversalFastHTTPHandlerOpts[*runtimev1pb.SubtleDecryptAlpha1Request, *runtimev1pb.SubtleDecryptAlpha1Response]{
+			InModifier: subtleCryptoInModifier[*runtimev1pb.SubtleDecryptAlpha1Request],
+		},
+	)
+}
+
+func (a *api) onPostSubtleCryptoWrapKey() fasthttp.RequestHandler {
+	return UniversalFastHTTPHandler(
+		a.universal.SubtleWrapKeyAlpha1,
+		UniversalFastHTTPHandlerOpts[*runtimev1pb.SubtleWrapKeyAlpha1Request, *runtimev1pb.SubtleWrapKeyAlpha1Response]{
+			InModifier: subtleCryptoInModifier[*runtimev1pb.SubtleWrapKeyAlpha1Request],
+		},
+	)
+}
+
+func (a *api) onPostSubtleCryptoUnwrapKey() fasthttp.RequestHandler {
+	return UniversalFastHTTPHandler(
+		a.universal.SubtleUnwrapKeyAlpha1,
+		UniversalFastHTTPHandlerOpts[*runtimev1pb.SubtleUnwrapKeyAlpha1Request, *runtimev1pb.SubtleUnwrapKeyAlpha1Response]{
+			InModifier: subtleCryptoInModifier[*runtimev1pb.SubtleUnwrapKeyAlpha1Request],
+		},
+	)
+}
+
+func (a *api) onPostSubtleCryptoSign() fasthttp.RequestHandler {
+	return UniversalFastHTTPHandler(
+		a.universal.SubtleSignAlpha1,
+		UniversalFastHTTPHandlerOpts[*runtimev1pb.SubtleSignAlpha1Request, *runtimev1pb.SubtleSignAlpha1Response]{
+			InModifier: subtleCryptoInModifier[*runtimev1pb.SubtleSignAlpha1Request],
+		},
+	)
```

Comment:
@ItalyPaleAle adding unit tests for these apis would be good

Reply:
When they get enabled, yes :) Right now, there's not much to test